Docs: uv-based lint command checks artifact directory

### Bug Description

When @jahn-junior and I try and run `make lint-docs`, which calls the newer uv profile for linting, it includes the `_build` directory in the check, resulting in hundreds of errors.

Other tests:

- Starbase's uv config doesn't reproduce this issue. It uses sphinx-lint 1.0.0, while Charmcraft uses 0.9.2, but upgrading the dependency in Charmcraft made no difference.
- Adding another `--ignore` or `-i` flag pointing to `docs/_build` fixed the issue, but I'm not sure why the dir is included in the first place.

There might be something very obvious that we're missing.

### To Reproduce

1. Run `make docs`.
2. Run `make lint-docs`.

### Environment

Ubuntu 24.04

### charmcraft.yaml

```yaml
-
```

### Relevant log output

```shell
Makefile:40: warning: overriding recipe for target 'setup'
common.mk:45: warning: ignoring old recipe for target 'setup'
uv run --extra docs sphinx-lint --max-line-length 88 -i docs/reference/commands --enable all docs
docs/_build/tutorial/write-your-first-kubernetes-charm-for-a-django-app.html:596: possibly leaked markup: <span class="n">ALLOWED_HOSTS</span> <span class="o">=</span> <span class="n">json</span><span class="o">.</span><span class="n">loads</span><span class="p">(</span><span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s1">&#39;DJANGO_ALLOWED_HOSTS&#39;</span><span class="p">,</span> <span class="s1">&#39;</span><span class="si">{ref}</span><span class="s1">`]&#39;</span><span class="p">))</span>
 (leaked-markup)
docs/_build/reference/changelog.html:10: possibly leaked markup: <meta property="og:description" content="3.3.0 (2025-01-13): Core: Plugins: New Poetry plugin., New Python plugin.. Reactive: Extensions: New fastapi-framework extension, The Django extension is no longer considered experimental., The cha..." />
 (leaked-markup)
docs/_build/reference/changelog.html:13: possibly leaked markup: <meta name="description" content="3.3.0 (2025-01-13): Core: Plugins: New Poetry plugin., New Python plugin.. Reactive: Extensions: New fastapi-framework extension, The Django extension is no longer considered experimental., The cha..." />
 (leaked-markup)
docs/_build/reference/commands/fetch-libs.html:426: possibly leaked markup: <span class="c1"># If `fetch-libs` is run and a newer minor version is available,</span>
 (leaked-markup)
docs/_build/reference/extensions/charmcraft-extension-django-framework.html:489: possibly leaked markup: In addition to these integrations, in each `provides` and `requires` block you may specify further integration endpoints, to integrate with the following charms and bundles:
 (leaked-markup)
docs/_build/reference/extensions/charmcraft-extension-django-framework.html:505: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/extensions/charmcraft-extension-fastapi-framework.html:478: possibly leaked markup: In addition to these integrations, in each `provides` and `requires` block you may specify further integration endpoints, to integrate with the following charms and bundles:
 (leaked-markup)
docs/_build/reference/extensions/charmcraft-extension-fastapi-framework.html:493: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/extensions/charmcraft-extension-go-framework.html:485: possibly leaked markup: In addition to these, in each `provides` and `requires` block you may specifying further integration endpoints, to integrate with the following charms and bundles:
 (leaked-markup)
docs/_build/reference/extensions/charmcraft-extension-go-framework.html:503: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/extensions/charmcraft-extension-flask-framework.html:478: possibly leaked markup: In addition to these, in each `provides` and `requires` block you may specifying further integration endpoints, to integrate with the following charms and bundles:
 (leaked-markup)
docs/_build/reference/extensions/charmcraft-extension-flask-framework.html:496: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:476: possibly leaked markup: <p>On the Juju end, profiles are upgraded during <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">refresh</span> <span class="pre">&lt;charm&gt;</span></code>; applied automatically during <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">deploy</span> <span class="pre">&lt;charm&gt;;</span> <span class="pre">and</span> <span class="pre">displayed</span> <span class="pre">at</span> <span class="pre">the</span> <span class="pre">machine</span> <span class="pre">level</span> <span class="pre">via</span> </code>juju show-machine<code class="docutils literal notranslate"><span class="pre">or</span></code>juju status –format=yaml`.</p>
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:480: possibly leaked markup: - Profiles are upgraded during the upgrade of the charm (`juju refresh <charm>`).
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:481: possibly leaked markup: - Profiles are displayed at the machine level by using either the `show-machine` command or the `status --format=yaml` command. Below is an example of the kind of information that can be obtained from either of these two commands:
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:483: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:488: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:491: possibly leaked markup: Juju (`v.2.5.0`) supports LXD profiles for charms. This is implemented by including file `lxd-profile.yaml` in a  charm's root directory. For example, here is a simple two-line file (this is taken from the [Openvswitch](https://jaas.ai/neutron-openvswitch) charm):
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:493: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:496: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:498: possibly leaked markup: - A validity check is performed on the profile(s) during the deployment of the charm. This is based on a hardcoded list of allowed items, everything else being denied. The `--force` option can be used to bypass this check but this is not recommended. The list is:
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:500: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-lxd-profile-yaml.html:511: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/plugins/poetry_plugin.html:431: possibly leaked markup: <p>Extra arguments to pass at the end of the poetry <a href="#id2"><span class="problematic" id="id3">`export command`_</span></a>.</p>
 (leaked-markup)
docs/_build/reference/files/file-charmcraft-yaml.html:830: possibly leaked markup: <span class="c1"># Accepts all runtime bases and ``ubuntu@devel``</span>
 (leaked-markup)
docs/_build/reference/files/file-charmcraft-yaml.html:1329: possibly leaked markup: <p class="sd-card-text">..code:</p>
 (leaked-markup)
docs/_build/reference/files/file-charmcraft-yaml.html:1399: possibly leaked markup: <p class="sd-card-text">..code:</p>
 (leaked-markup)
docs/_build/reference/files/charmcraft-yaml-file.html:773: possibly leaked markup: <span class="c1"># Accepts all runtime bases and ``ubuntu@devel``</span>
 (leaked-markup)
docs/_build/reference/files/actions-yaml.html:348: possibly leaked markup: <li class="toctree-l3 current current-page"><a class="current reference internal" href="#">``actions.yaml``</a></li>
 (leaked-markup)
docs/_build/reference/files/file-bundle-yaml.html:1192: possibly leaked markup: <p class="sd-card-text">..code:</p>
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:570: possibly leaked markup: <span class="c1"># will ultimately use to deploy the charm. E.g. `juju deploy &lt;name&gt;`</span>
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:862: possibly leaked markup: <tr class="row-even"><td><p><code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">&lt;comparison</span> <span class="pre">predicate&gt;</span> <span class="pre">&lt;version</span> <span class="pre">number&gt;</span></code> <br><br> E.g., <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">&lt;</span> <span class="pre">3.0</span></code>. <br> E.g., juju &gt;= 2.9`</p></td>
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:958: possibly leaked markup: <span class="w">        </span><span class="nt">channel</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;track{ref}`/risk][/branch]&gt;</span>
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:969: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1023: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1050: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1054: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1122: possibly leaked markup: <span class="nt">issues</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;url&gt; | {ref}`&lt;urls&gt;]</span>
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1142: possibly leaked markup: <span class="c1"># administrators will ultimately use to deploy the charm. E.g. `juju deploy &lt;name&gt;`</span>
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1222: possibly leaked markup: The `resources` block consists
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1225: possibly leaked markup: |    `type`     | `string` | `file`  | Type of the resource. Supported values are `file` or `oci-image`. |
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1226: possibly leaked markup: | `description` | `string` |  `nil`  | Description of the resource                                       |
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1227: possibly leaked markup: |  `filename`   | `string` |  `nil`  | Name of the file resource                                         |
 (leaked-markup)
docs/_build/reference/files/file-metadata-yaml.html:1266: possibly leaked markup: <span class="nt">source</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;url&gt; | {ref}`&lt;urls&gt;]</span>
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:441: possibly leaked markup: <p>On the Juju end, profiles are upgraded during <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">refresh</span> <span class="pre">&lt;charm&gt;</span></code>; applied automatically during <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">deploy</span> <span class="pre">&lt;charm&gt;;</span> <span class="pre">and</span> <span class="pre">displayed</span> <span class="pre">at</span> <span class="pre">the</span> <span class="pre">machine</span> <span class="pre">level</span> <span class="pre">via</span> </code>juju show-machine<code class="docutils literal notranslate"><span class="pre">or</span></code>juju status –format=yaml`.</p>
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:445: possibly leaked markup: - Profiles are upgraded during the upgrade of the charm (`juju refresh <charm>`).
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:446: possibly leaked markup: - Profiles are displayed at the machine level by using either the `show-machine` command or the `status --format=yaml` command. Below is an example of the kind of information that can be obtained from either of these two commands:
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:448: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:453: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:456: possibly leaked markup: Juju (`v.2.5.0`) supports LXD profiles for charms. This is implemented by including file `lxd-profile.yaml` in a  charm's root directory. For example, here is a simple two-line file (this is taken from the [Openvswitch](https://jaas.ai/neutron-openvswitch) charm):
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:458: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:461: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:463: possibly leaked markup: - A validity check is performed on the profile(s) during the deployment of the charm. This is based on a hardcoded list of allowed items, everything else being denied. The `--force` option can be used to bypass this check but this is not recommended. The list is:
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:465: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml.html:476: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/libname-py-file.html:480: possibly leaked markup: <p><strong>Value:</strong> <code class="docutils literal notranslate"><span class="pre">LIBAPI`</span></code> is set to an initial state of <code class="docutils literal notranslate"><span class="pre">0</span></code>. In general,
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:528: possibly leaked markup: <span class="c1"># will ultimately use to deploy the charm. E.g. `juju deploy &lt;name&gt;`</span>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:793: possibly leaked markup: <tr class="row-even"><td><p><code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">&lt;comparison</span> <span class="pre">predicate&gt;</span> <span class="pre">&lt;version</span> <span class="pre">number&gt;</span></code> <br> E.g., <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">&lt;</span> <span class="pre">3.0</span></code>. <br> E.g., juju &gt;= 2.9`</p></td>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:889: possibly leaked markup: <span class="w">        </span><span class="nt">channel</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;track{ref}`/risk][/branch]&gt;</span>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:900: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:954: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:981: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:985: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:1052: possibly leaked markup: <span class="nt">issues</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;url&gt; | {ref}`&lt;urls&gt;]</span>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:1071: possibly leaked markup: <span class="c1"># administrators will ultimately use to deploy the charm. E.g. `juju deploy &lt;name&gt;`</span>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:1151: possibly leaked markup: The `resources` block consists
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:1154: possibly leaked markup: |    `type`     | `string` | `file`  | Type of the resource. Supported values are `file` or `oci-image`. |
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:1155: possibly leaked markup: | `description` | `string` |  `nil`  | Description of the resource                                       |
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:1156: possibly leaked markup: |  `filename`   | `string` |  `nil`  | Name of the file resource                                         |
 (leaked-markup)
docs/_build/reference/files/metadata-yaml-file.html:1195: possibly leaked markup: <span class="nt">source</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;url&gt; | {ref}`&lt;urls&gt;]</span>
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:434: possibly leaked markup: <p>On the Juju end, profiles are upgraded during <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">refresh</span> <span class="pre">&lt;charm&gt;</span></code>; applied automatically during <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">deploy</span> <span class="pre">&lt;charm&gt;;</span> <span class="pre">and</span> <span class="pre">displayed</span> <span class="pre">at</span> <span class="pre">the</span> <span class="pre">machine</span> <span class="pre">level</span> <span class="pre">via</span> </code>juju show-machine<code class="docutils literal notranslate"><span class="pre">or</span></code>juju status –format=yaml`.</p>
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:438: possibly leaked markup: - Profiles are upgraded during the upgrade of the charm (`juju refresh <charm>`).
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:439: possibly leaked markup: - Profiles are displayed at the machine level by using either the `show-machine` command or the `status --format=yaml` command. Below is an example of the kind of information that can be obtained from either of these two commands:
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:441: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:446: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:449: possibly leaked markup: Juju (`v.2.5.0`) supports LXD profiles for charms. This is implemented by including file `lxd-profile.yaml` in a  charm's root directory. For example, here is a simple two-line file (this is taken from the [Openvswitch](https://jaas.ai/neutron-openvswitch) charm):
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:451: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:454: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:456: possibly leaked markup: - A validity check is performed on the profile(s) during the deployment of the charm. This is based on a hardcoded list of allowed items, everything else being denied. The `--force` option can be used to bypass this check but this is not recommended. The list is:
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:458: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/lxd-profile-yaml-file.html:469: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/charmcraft-yaml.html:800: possibly leaked markup: <span class="c1"># Accepts all runtime bases and ``ubuntu@devel``</span>
 (leaked-markup)
docs/_build/reference/files/libname-py.html:493: possibly leaked markup: <p><strong>Value:</strong> <code class="docutils literal notranslate"><span class="pre">LIBAPI`</span></code> is set to an initial state of <code class="docutils literal notranslate"><span class="pre">0</span></code>. In general,
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:535: possibly leaked markup: <span class="c1"># will ultimately use to deploy the charm. E.g. `juju deploy &lt;name&gt;`</span>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:827: possibly leaked markup: <tr class="row-even"><td><p><code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">&lt;comparison</span> <span class="pre">predicate&gt;</span> <span class="pre">&lt;version</span> <span class="pre">number&gt;</span></code> <br><br> E.g., <code class="docutils literal notranslate"><span class="pre">juju</span> <span class="pre">&lt;</span> <span class="pre">3.0</span></code>. <br> E.g., juju &gt;= 2.9`</p></td>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:923: possibly leaked markup: <span class="w">        </span><span class="nt">channel</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;track{ref}`/risk][/branch]&gt;</span>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:934: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:988: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1015: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1019: possibly leaked markup: 
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1087: possibly leaked markup: <span class="nt">issues</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;url&gt; | {ref}`&lt;urls&gt;]</span>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1107: possibly leaked markup: <span class="c1"># administrators will ultimately use to deploy the charm. E.g. `juju deploy &lt;name&gt;`</span>
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1187: possibly leaked markup: The `resources` block consists
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1190: possibly leaked markup: |    `type`     | `string` | `file`  | Type of the resource. Supported values are `file` or `oci-image`. |
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1191: possibly leaked markup: | `description` | `string` |  `nil`  | Description of the resource                                       |
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1192: possibly leaked markup: |  `filename`   | `string` |  `nil`  | Name of the file resource                                         |
 (leaked-markup)
docs/_build/reference/files/metadata-yaml.html:1231: possibly leaked markup: <span class="nt">source</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">&lt;url&gt; | {ref}`&lt;urls&gt;]</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/charmcraft.html:464: possibly leaked markup: <span class="w">    </span><span class="sd">&quot;&quot;&quot;Definition of `analysis.ignore` configuration.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/charmcraft.html:471: possibly leaked markup: <span class="w">    </span><span class="sd">&quot;&quot;&quot;Definition of `analysis` configuration.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/charmcraft.html:479: possibly leaked markup: <span class="w">    </span><span class="sd">&quot;&quot;&quot;Definition of `links` in metadata.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/reference/files/file-libname-py.html:522: possibly leaked markup: <p><strong>Value:</strong> <code class="docutils literal notranslate"><span class="pre">LIBAPI`</span></code> is set to an initial state of <code class="docutils literal notranslate"><span class="pre">0</span></code>. In general,
 (leaked-markup)
docs/_build/howto/set-up-charmcraft.html:472: possibly leaked markup: <code class="output docutils literal notranslate"><span class="pre">Multipass</span> <span class="pre">is</span> <span class="pre">required,</span> <span class="pre">but</span> <span class="pre">not</span> <span class="pre">installed.</span> <span class="pre">Do</span> <span class="pre">you</span> <span class="pre">wish</span> <span class="pre">to</span> <span class="pre">install</span> <span class="pre">Multipass</span> <span class="pre">and</span> <span class="pre">configure</span> <span class="pre">it</span> <span class="pre">with</span> <span class="pre">the</span> <span class="pre">defaults?</span> <span class="pre">[y/N]:</span> <span class="pre">y</span></code><code class="output docutils literal notranslate"><span class="pre">==&gt;</span> <span class="pre">Downloading</span> <span class="pre">https://github.com/canonical/multipass/releases/download/v1.7.2/multipass-1.7.2+mac-Darwin.pkg</span></code><code class="output docutils literal notranslate"><span class="pre">Already</span> <span class="pre">downloaded:</span> <span class="pre">/Users/jdoe/Library/Caches/Homebrew/downloads/4237fcef800faa84459a2911c3818dfa76f1532d693b151438f1c8266318715b--multipass-1.7.2+mac-Darwin.pkg</span></code><code class="output docutils literal notranslate"><span class="pre">==&gt;</span> <span class="pre">Installing</span> <span class="pre">Cask</span> <span class="pre">multipass</span></code><code class="output docutils literal notranslate"><span class="pre">==&gt;</span> <span class="pre">Running</span> <span class="pre">installer</span> <span class="pre">for</span> <span class="pre">multipass;</span> <span class="pre">your</span> <span class="pre">password</span> <span class="pre">may</span> <span class="pre">be</span> <span class="pre">necessary.</span></code><code class="output docutils literal notranslate"><span class="pre">Package</span> <span class="pre">installers</span> <span class="pre">may</span> <span class="pre">write</span> <span class="pre">to</span> <span class="pre">any</span> <span class="pre">location;</span> <span class="pre">options</span> <span class="pre">such</span> <span class="pre">as</span> <span class="pre">`--appdir`</span> <span class="pre">are</span> <span class="pre">ignored.</span></code><code class="output docutils literal notranslate"><span class="pre">installer:</span> <span class="pre">Package</span> <span class="pre">name</span> <span class="pre">is</span> <span class="pre">multipass</span></code><code class="output docutils literal notranslate"><span class="pre">installer:</span> <span class="pre">Installing</span> <span class="pre">at</span> <span class="pre">base</span> <span class="pre">path</span> <span class="pre">/</span></code><code class="output docutils literal notranslate"><span class="pre">installer:</span> <span class="pre">The</span> <span class="pre">install</span> <span class="pre">was</span> <span class="pre">successful.</span></code><code class="output docutils literal notranslate"><span class="pre">🍺</span>&#160; <span class="pre">multipass</span> <span class="pre">was</span> <span class="pre">successfully</span> <span class="pre">installed!</span></code><code class="output docutils literal notranslate"><span class="pre">Packing</span> <span class="pre">charm</span> <span class="pre">'test-charm_ubuntu-20.04-amd64.charm'...</span></code><code class="output docutils literal notranslate"><span class="pre">Starting</span> <span class="pre">charmcraft-test-charm-12886917363-0-0-amd64</span> <span class="pre">...</span></code></div>
 (leaked-markup)
docs/_build/howto/manage-a-12-factor-app-charm.html:426: possibly leaked markup: See {ref}`Rockcraft Documentation > How to update the OCI image... <15018md>`
 (leaked-markup)
docs/_build/howto/manage-a-12-factor-app-charm.html:541: possibly leaked markup: </details><!--In addition to the environment variable, the configuration is also available in the FastAPI variable `app.config` without the `APP_` prefix.-->
 (leaked-markup)
docs/_build/howto/manage-channels.html:439: possibly leaked markup: When you register, you get a track called `latest` with all the usual risk levels. So, you get all of:
 (leaked-markup)
docs/_build/howto/manage-channels.html:464: possibly leaked markup: <code class="output docutils literal notranslate"> <span class="pre">Track</span>&#160;&#160;&#160; <span class="pre">Channel</span>&#160;&#160;&#160; <span class="pre">Version</span>&#160;&#160;&#160; <span class="pre">Revision</span></code><code class="output docutils literal notranslate"> <span class="pre">latest</span>&#160;&#160; <span class="pre">stable</span>&#160;&#160;&#160;&#160; <span class="pre">-</span>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">-</span></code><code class="output docutils literal notranslate">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">candidate</span>&#160; <span class="pre">-</span>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">-</span></code><code class="output docutils literal notranslate">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">beta</span>&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">0.1</span>&#160;&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">1</span></code><code class="output docutils literal notranslate">&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">edge</span>&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">↑</span>&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160; <span class="pre">↑</span></code><code class="output docutils literal notranslate"> </code><code class="output docutils literal notranslate"> </code><code class="output docutils literal notranslate"><span class="pre">See</span> <span class="pre">more:</span> <span class="pre">:ref:`ref_commands_status`</span></code></div>
 (leaked-markup)
docs/_build/howto/manage-the-current-charmhub-user.html:507: possibly leaked markup: <div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="nv">CHARMCRAFT_AUTH</span><span class="o">=</span><span class="sb">`</span>cat<span class="w"> </span>test1<span class="sb">`</span>
 (leaked-markup)
docs/_build/howto/manage-the-charmcraft-cli.html:522: possibly leaked markup: Package installers may write to any location; options such as `--appdir` are ignored.
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:744: possibly leaked markup: <span class="sd">        `CharmBuildInfo.gen_from_bases_configurations&#39;. Otherwise, it generates the BuildInfo</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:974: possibly leaked markup: <span class="w">    </span><span class="sd">&quot;&quot;&quot;The type of project. Must be the string ``charm``.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:984: possibly leaked markup: <span class="sd">            `charm naming guidelines &lt;https://juju.is/docs/sdk/naming&gt;`_.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1005: possibly leaked markup: <span class="sd">            `actions.yaml file &lt;https://juju.is/docs/sdk/actions-yaml&gt;`_.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1019: possibly leaked markup: <span class="sd">            ``juju status`` to figure out whether the deployment was successful.</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1039: possibly leaked markup: <span class="sd">            ``juju &gt;= 3.5`` or ``juju &lt; 4.0``. A full list of supported features</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1041: possibly leaked markup: <span class="sd">            `Juju documentation &lt;https://juju.is/docs/juju/supported-features&gt;`_.</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1052: possibly leaked markup: <span class="sd">            - ``root``: the charm will run as root</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1053: possibly leaked markup: <span class="sd">            - ``sudoer``: the charm will run as a non-root user with access to root</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1054: possibly leaked markup: <span class="sd">              privileges using ``sudo``.</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1055: possibly leaked markup: <span class="sd">            - ``non-root``: the charm will run as a non-root user without ``sudo``.</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1059: possibly leaked markup: <span class="sd">            `its default behaviour &lt;https://juju.is/docs/sdk/metadata-yaml#heading--charm-user&gt;`_.</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1073: possibly leaked markup: <span class="sd">            specifications. Each container can be specified in terms of ``resource``,</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1074: possibly leaked markup: <span class="sd">            ``bases`` and ``mounts``, where one of either the ``resource`` or the</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1075: possibly leaked markup: <span class="sd">            ``bases`` subkeys must be defined and ``mounts`` is optional.</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1077: possibly leaked markup: <span class="sd">            - ``resource`` is the name of an OCI image resource used to create the</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1079: possibly leaked markup: <span class="sd">            - ``bases`` is a list of bases to be used for resolving a container image,</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1081: possibly leaked markup: <span class="sd">              example, ``ubuntu`` or ``centos``), a ``channel`` and an</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1082: possibly leaked markup: <span class="sd">              ``architecture``.</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1083: possibly leaked markup: <span class="sd">            - ``mounts`` is a list of mounted storage volumes for this container. To</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1169: possibly leaked markup: <span class="sd">            `the Juju documentation &lt;https://juju.is/docs/juju/relation&gt;`_.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1198: possibly leaked markup: <span class="sd">            `the Juju documentation &lt;https://juju.is/docs/juju/relation&gt;`_.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1218: possibly leaked markup: <span class="sd">            `the Juju documentation &lt;https://juju.is/docs/juju/relation&gt;`_.&quot;&quot;&quot;</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1237: possibly leaked markup: <span class="sd">            See first: `Juju | Charm resource &lt;https://juju.is/docs/juju/charm-resource&gt;`_</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1240: possibly leaked markup: <span class="sd">            Kubernetes charms must declare an ``oci-image`` type resource for each</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1241: possibly leaked markup: <span class="sd">            container declared in ``containers``.</span>
 (leaked-markup)
docs/_build/_modules/charmcraft/models/project.html:1429: possibly leaked markup: <span class="w">    </span><span class="sd">&quot;&quot;&quot;A charm using the deprecated ``bases`` keyword.</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:440: possibly leaked markup: <span class="w">   </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;A charm using the deprecated ``bases`` keyword.\n\nThis type of charm only supports the following bases:\n    - Ubuntu 18.04\n    - Ubuntu 20.04\n    - Ubuntu 22.04\n    - CentOS 7\n    - Alma Linux 9&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:444: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;The name of the project on Charmhub.\n\nThis value will be used both in the URL of the charm on Charmhub and when\ndeploying the charm with juju.\n\nCharms should follow the\n`charm naming guidelines &lt;https://juju.is/docs/sdk/naming&gt;`_.&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:632: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Defines one or more actions.\n\nThis key is equivalent to the\n`actions.yaml file &lt;https://juju.is/docs/sdk/actions-yaml&gt;`_.&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:666: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Explicitly state features a Juju model must be able to provide for a\nsuccessful deployment of this charm. When a charm includes such\nrequirements, Juju performs a pre-deployment check and displays\nuser-friendly error messages if a feature requirement cannot be met by the\nmodel that the user is trying to deploy the charm to. If the assumes\nsection of the charm metadata is omitted, Juju will make a best-effort\nattempt to deploy the charm, and users must rely on the output of\n``juju status`` to figure out whether the deployment was successful.\n\nThe key consists of a list of features that can be given either directly\nor, depending on the complexity of the condition you want to enforce,\nnested under one or both of the boolean expressions any-of or all-of,\nas shown below. In order for a charm to be deployed, all entries in the\nassumes block must be satisfied.\n\nStructure::\n\n    assumes:\n      - &lt;feature-1&gt;\n      - any-of:\n        - &lt;feature-2&gt;\n        - &lt;feature-3&gt;\n      - all-of:\n        - &lt;feature-4&gt;\n        - &lt;feature-5&gt;\n\nJuju version requirements can be specified with a string such as\n``juju &gt;= 3.5`` or ``juju &lt; 4.0``. A full list of supported features\ncan be found in the\n`Juju documentation &lt;https://juju.is/docs/juju/supported-features&gt;`_.\n&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:684: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Specifies that the charm code does not need to be run as root. Possible\nvalues are:\n\n- ``root``: the charm will run as root\n- ``sudoer``: the charm will run as a non-root user with access to root\n  privileges using ``sudo``.\n- ``non-root``: the charm will run as a non-root user without ``sudo``.\n\nOnly affects Kubernetes charms on Juju 3.6.0 or later. If not specified,\nJuju will use\n`its default behaviour &lt;https://juju.is/docs/sdk/metadata-yaml#heading--charm-user&gt;`_.\n&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:697: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Define a map of containers to be created adjacent to the charm (as a\nsidecar, in the same pod).\n\nThis is required for Kubernetes charms.\n\nThis key consists of a dictionary mapping container names to their\nspecifications. Each container can be specified in terms of ``resource``,\n``bases`` and ``mounts``, where one of either the ``resource`` or the\n``bases`` subkeys must be defined and ``mounts`` is optional.\n\n- ``resource`` is the name of an OCI image resource used to create the\n  container (that you will then define further in the resources block).\n- ``bases`` is a list of bases to be used for resolving a container image,\n  in descending order of preference. To use it, specify a base name (for\n  example, ``ubuntu`` or ``centos``), a ``channel`` and an\n  ``architecture``.\n- ``mounts`` is a list of mounted storage volumes for this container. To\n  use it, specify the name of the storage to mount from the charm\n  storage and, optionally, the location where to mount the storage.\n\nStructure::\n\n    containers:\n      &lt;container name&gt;:\n        resource: &lt;resource name&gt;\n        bases:\n          - name: &lt;base name&gt;\n            channel: &lt;track[/risk][/branch]&gt;\n            architectures:\n              - &lt;architecture&gt;\n        mounts:\n          - storage: &lt;storage name&gt;\n            location: &lt;path&gt;&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:772: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;A map of peer relations.\n\nStructure::\n\n    peers:\n      &lt;endpoint name&gt;:\n        interface: &lt;Required interface name&gt;\n        limit: &lt;Optional: maximum number of supported connections\n        optional: &lt;Informational only - whether the relation is required.&gt;\n        scope: &lt;\&quot;global\&quot; or \&quot;container\&quot; - the relation scope.&gt;\n\nFor more information, see\n`the Juju documentation &lt;https://juju.is/docs/juju/relation&gt;`_.&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:795: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;A map of interfaces this charm provides.\n\nStructure::\n\n    provides:\n      &lt;endpoint name&gt;:\n        interface: &lt;Required interface name&gt;\n        limit: &lt;Optional: maximum number of supported connections\n        optional: &lt;Informational only - whether the relation is required.&gt;\n        scope: &lt;\&quot;global\&quot; or \&quot;container\&quot; - the relation scope.&gt;\n\nFor more information, see\n`the Juju documentation &lt;https://juju.is/docs/juju/relation&gt;`_.&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:815: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;A map of relations this charm requires.\n\nStructure::\n\n    requires:\n      &lt;endpoint name&gt;:\n        interface: &lt;Required interface name&gt;\n        limit: &lt;Optional: maximum number of supported connections\n        optional: &lt;Informational only - whether the relation is required.&gt;\n        scope: &lt;\&quot;global\&quot; or \&quot;container\&quot; - the relation scope.&gt;\n\nFor more information, see\n`the Juju documentation &lt;https://juju.is/docs/juju/relation&gt;`_.&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:838: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;A mapping of resources that accompany the charm.\n\nSee first: `Juju | Charm resource &lt;https://juju.is/docs/juju/charm-resource&gt;`_\n\nEach resource is made available when the charm is deployed. NOTE:\nKubernetes charms must declare an ``oci-image`` type resource for each\ncontainer declared in ``containers``.\n\nStructure::\n\n    # (Optional) Additional resources that accompany the charm\n    resources:\n      &lt;resource name&gt;:\n        # (Required) The type of the resource\n        type: file | oci-image\n\n        # (Optional) Description of the resource and its purpose\n        description: &lt;description&gt;\n\n        # (Required: when type:file) The filename of the resource as it\n        # should appear in the filesystem.\n        filename: &lt;filename&gt;&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:978: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Definition of `analysis` configuration.&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:1103: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Definition of `analysis.ignore` configuration.&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:1127: possibly leaked markup: <span class="w">     </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Definition of `links` in metadata.&quot;</span><span class="p">,</span>
 (leaked-markup)
docs/_build/reference/models/bases-charm.html:1506: possibly leaked markup: <span class="w">   </span><span class="nt">&quot;description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Definition of `links` in metadata.&quot;</span><span class="p">,</span>
 (leaked-markup)
make: *** [common.mk:157: lint-docs] Error 1
```

--- a/common.mk
+++ b/common.mk
@@ -154,7 +154,7 @@ lint-docs:  ##- Lint the documentation
 ifneq ($(CI),)
 	@echo ::group::$@
 endif
-	uv run --extra docs sphinx-lint --max-line-length 88 --ignore docs/reference/commands --enable all $(DOCS)
+	uv run --extra docs sphinx-lint --max-line-length 88 --ignore docs/reference/commands --ignore docs/_build/ --enable all $(DOCS)
 ifneq ($(CI),)
 	@echo ::endgroup::
 endif

